### DISCUSSION TITLE
Is the API rate limit for the same seller shared or independent among different authorized applications?

### DISCUSSION THREAD

**Discussion by kelerensheng** [amzn/selling-partner-api-models#3678] [General] [answered]
As doc [description](https://developer-docs.amazon.com/sp-api/docs/usage-plans-and-rate-limits-in-the-sp-api#will-my-application-be-throttled-more-often-as-i-obtain-more-authorizations): All usage plans are specific to application-Selling Partner pairs, so that your throughput grows naturally with your clients.
But I would like to know, for the same seller, if it authorizes two different apps, will the API speed limits be shared between the two apps, or will each have a separate speed limit?

**Comment by leonardonakagawa** [comment, marked as answer]
Hi @kelerensheng,

For all APIs the Rate Limit works accordingly to the App, Seller, and Operation levels except for Feeds and Reports where the rate limit could be shared with more than one application the seller has authorized. 

Please also take into consideration that for most questions, our documentation and frequently asked questions can be found here (https://developer-docs.amazon.com/sp-api/).

If this is a technical issue that requires additional troubleshooting steps because you are getting any kind of error response from the API, please open a support case instead (https://developer.amazonservices.com/support).

Thanks,
Leonardo Nakagawa
Selling Partner API Developer Support

**Reply by kelerensheng** [reply]
Hi @leonardonakagawa ,

Thanks for your reply. As you mentioned, I did carefully read the api document before, and also searched for issues and discussions, but apart from the description of rate limit I mentioned above, I did not find the description of whether different apps share rate limit.

But now I can confirm that except Feed and Reports, other operations will not share rate limt between apps.

Thanks again!

**Comment by mpenna** [comment]
This question was the closest I could find in dealing with the sort of information I'm looking for. Therefore, I apologize piggy-backing on another person's thread. 

I have an application where eventually many client instances spawned with the same app credentials (client id, secret, etc.) but having each their own auth tokens created through the use of the same seller's refresh token could be requesting data concurrently from the same operation endpoint. In such a situation, will they all be considered in aggregate towards said operation's burst and rate limit specifications or will each instance be individually considered for such usage?
